drivers/i2c/Kconfig

```diff
@@ -107,6 +107,13 @@ config I2C_MCUX_LPI2C
 	help
 	  Enable the mcux LPI2C driver.
 
+config I2C_MCUX_LPI2C_BUS_RECOVERY
+	bool "Bus recovery support"
+	depends on I2C_MCUX_LPI2C && PINCTRL
+	select I2C_BITBANG
+	help
+	  Enable LPI2C driver bus recovery support via GPIO bitbanging.
+
```

Comment:
I guess this could be derived from the devicetree as well (if a node has the `scl-gpios` and `sda-gpios` properties and the correct `compatible` we enable the code). What do people think? Kconfig or automatic derived from devicetree?

Reply:
Would the dts for most dev kits have these properties added for their default expansion port i2c pinmux?  If it was automatic, there wouldn't be a good way to turn it off.

Reply:
The only way to turn it off would then be using an overlay for removing those properties.